### PR TITLE
HCLIND-8 Update: Replace whitelist term with allowed list for Azure policies

### PR DESCRIPTION
**Pull request by juanestrahcl**
### Description

Replaced **whitelist** term with **allowed list** in Azure related files.

### Issues Resolved

From HMRC: _Terminology of “whitelisted” sounds and reads racists_.

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Review by taniaSol — COMMENTED** [review]
Looks good.
I test one policy - Azure Idle Compute Instances and it ran fine

**Review by taterdog590 — APPROVED** [review]
Looks good to me.

**Review by degarajesh — COMMENTED** [review]
LGTM

File: security/azure/webapp_tls_version_support/azure_webapp_min_tls_version.pt [hunk 1]
```diff
@@ -7,7 +7,7 @@ severity "low"
 category "Security"
 default_frequency "daily"
 info(
-  version: "2.4",
+  version: "2.6",
```
Comment (degarajesh) [inline security/azure/webapp_tls_version_support/azure_webapp_min_tls_version.pt:10]: Ideally master should have 2.5, I see change change.log 2.5 in the policy it is not updated it. However Juan is updating correct version.

**Pull request merged**